src/status_im/subs/activity_center.cljs

```diff
@@ -29,6 +29,7 @@
  :<- [:activity-center/filter-status]
  :<- [:activity-center/notifications]
  (fn [[filter-type filter-status notifications]]
+   (println "asdfasdf" filter-type filter-status)
```

Comment:
The famous speed typing test :1st_place_medal: 
 https://10fastfingers.com/text/8258-ASDF-Practice

Reply:
😂😂